### PR TITLE
update readme disclaimer

### PR DESCRIPTION
**Pull request by huttmf**
Moved disclaimer to top.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -2,7 +2,13 @@
 AOSOM-Streaming
 ###############
 
+**************
 Version v0.6.1
```
Comment (Ragsboss) [inline README.rst]: Previously this line was normal text - now it's a heading. Is this intentional?

File: README.rst [hunk 2]
```diff
@@ -2,7 +2,13 @@
...
+**************
+
+Disclaimer
+==========
+WARNING: This application is for demo purposes only and should not be used in production.
```
Comment (Ragsboss) [inline README.rst]: Put WARNING in bold

File: README.rst [hunk 3]
```diff
@@ -2,7 +2,13 @@
...
 
 The Aosom Streaming project provides a packaged solution to collect
```
Comment (Ragsboss) [inline README.rst]: Add an overview as heading at the same level as Disclaimer - right now it looks like all of this is in disclaimer.

File: README.rst (hunk 1 above)
Comment (huttmf) [reply README.rst]: Yes

**Pull request merged**